resources/docs/livewire/installation.md

```diff
@@ -0,0 +1,176 @@
+
+By default, the installer will pre-configure Laravel Sail with a number of useful services for your application, including a MySQL database server. You may [customize the Sail services](https://laravel.com/docs/installation#choosing-your-sail-services) if needed.
+
+After the project has been created, you can navigate to the application directory and start Laravel Sail:
+
+```shell
+cd chirper
+
+./vendor/bin/sail up
+```
+
+> **Note**
+> You can [create a shell alias](https://laravel.com/docs/sail#configuring-a-shell-alias) that allows you execute Sail's commands more easily.
+
+ When developing applications using Sail, you may execute Artisan, NPM, and Composer commands via the Sail CLI instead of invoking them directly:
+
+```shell
+./vendor/bin/sail php --version
+./vendor/bin/sail artisan --version
+./vendor/bin/sail composer --version
+./vendor/bin/sail npm --version
+```
+
+Once the application's Docker containers have been started, you can access the application in your web browser at: [http://localhost](http://localhost).
+
+<img src="/img/screenshots/fresh.png" alt="A fresh Laravel installation" class="dark:hidden rounded-lg border shadow-lg" />
+<img src="/img/screenshots/fresh-dark.png" alt="A fresh Laravel installation" class="hidden dark:block rounded-lg border-gray-700 shadow-lg" />
+
+## Installing Livewire
+
+Now that we have a fresh Laravel application set up, we can install both Livewire and Livewire Volt. The latter is a simple Livewire extension that allows a Livewire component's PHP logic and Blade templates to coexist in the same file, similar to modern JavaScript frameworks like Vue and React.
+
+Open a new terminal in your `chirper` project directory and install Livewire with the given command:
+
+```shell
+composer require livewire/livewire livewire/volt
+
+php artisan volt:install
+```
+
+## Installing Laravel Breeze
+
+Next, we will give your application a head-start by installing [Laravel Breeze](https://laravel.com/docs/starter-kits#laravel-breeze), a minimal, simple implementation of all of Laravel's authentication features, including login, registration, password reset, email verification, and password confirmation. Once installed, you are welcome to customize the components to suit your needs.
+
+Laravel Breeze offers several options for your view layer, including Blade templates, or [Vue](https://vuejs.org/) and [React](https://reactjs.org/) with [Inertia](https://inertiajs.com/). Because Livewire uses Blade templates under the hood, we'll be using Blade for this tutorial.
+
+Open a new terminal in your `chirper` project directory and install your chosen stack with the given commands:
+
+```shell
+composer require laravel/breeze --dev
+
+php artisan breeze:install blade
+```
+
+Then, include the Livewire JavaScript and CSS assets in your application's new main Blade layout file (`resources/views/layouts/app.blade.php`) created by Breeze:
```

Comment:
It's a bit annoying that we need to add this manually as Livewire does it for you these days, but I guess we need to in this case so that Alpine is available on the other Breeze pages. Bring on the Livewire stack in Breeze! :grin:

Reply:
This will be fixed once we have the Livewire stack on breeze indeed.